process/statistics/statistics.go

```diff
@@ -13,18 +15,22 @@ const (
 	shardTpsDocIDPrefix = "shard"
 )
 
-var log = logger.GetOrCreate("indexer/process/generalInfo")
+var log = logger.GetOrCreate("indexer/process/statistics")
 
-type infoProcessor struct {
+type statisticsProcessor struct {
 }
 
-// NewGeneralInfoProcessor will create a new instance of general info processor
-func NewGeneralInfoProcessor() *infoProcessor {
-	return &infoProcessor{}
+// NewStatisticsProcessor will create a new instance of general statisticsProcessor
+func NewStatisticsProcessor() *statisticsProcessor {
+	return &statisticsProcessor{}
 }
 
-// PrepareGeneralInfo will prepare and general information about chain
-func (gip *infoProcessor) PrepareGeneralInfo(tpsBenchmark statistics.TPSBenchmark) (*data.TPS, []*data.TPS) {
+// PrepareStatistics will prepare and general information about chain
```

Comment:
`// PrepareStatistics will prepare the statistics about the chain`

Reply:
done.